### ISSUE TITLE
Cross-realm BGs not implemented

### ISSUE THREAD

**Issue by ghost** [vmangos/core#722]
**Current behavior**: Cross-realm BGs are not implemented.

**Expected behavior**: Cross-realm BG should be implemented (added in patch 1.12).

**Client version**: 1.12

**Commit hash**: c89abc4

**Operating system**: Any

**Comment by ratkosrb** [comment]
Unlikely to ever be implemented.

**Comment by ghost** [comment]
Why?

**Comment by ratkosrb** [comment]
Cause the core is built very differently than blizzard's servers. They are able to run individual maps on a separate server, and these map servers are able to talk to each other so that the player can be smoothly transferred from one server to the other, which is what makes the cross realm battlegrounds possible. Even the character selection screen is a separate entity that runs independently of the actual world. We have a standalone realm server that runs all of the maps and manages the whole world and all characters by itself, and does not interact with other realms in any way. You have to rewrite a giant portion of the project to implement cross realm functionality. Who is going to do that?

**Comment by ghost** [comment]
Oh, same for all other WoW servers I think… Why was such non-blizzlike behavior even implemented?

I'm leaving World of Warcraft until this code will be rewritten. Doing any changes in non-blizzlike code (unless it becomes blizzlike) has no sense.

**Comment by NickTyrer** [comment]
Can we close some of the PR's and Issues opened by this user?

**Comment by Elmsroth** [comment]
Ahahah .. is it a Troll ?

**Comment by GrenderG** [comment]
He created 2 different accounts to open Issues and now both are deleted. 🤣

**Comment by ghost** [comment]
Why did you close this issue?

Cross-realm BGs is 1.12 feature so it has to be implemented. Maybe someone will do it if you @Nolin7777 or @ratkosrb can't/don't know how.

**Comment by Gamemechanicwow** [comment]
> Why did you close this issue?
> 
> Cross-realm BGs is 1.12 feature so it has to be implemented. Maybe someone will do it if you @Nolin7777 or @ratkosrb can't/don't know how.

Even if someone wanted to do it - which i guess not - there is no reason to implement it. It would require alot of work for something that is very unlikely to see any practical use. This is almost as stupid as making an issue about mobs on gm island. 

Sorry dude. But there has been giving enough reasons to close this issue - as its not real issue - but more of a seperate project. Just keep it closed - unless you have something new to add...

**Comment by Gamemechanicwow** [comment]
ohh crap.. he already deleted his user again.

So let me rephrase.

This is almost as stupid as deleting your github account, if you cant stay the fuck away.

**Comment by GrenderG** [comment]
So annoying.
